BankManagerConsoleApp/BankManagerConsoleApp/main.swift

```diff
@@ -4,3 +4,7 @@
 //
 //  Created by kyungmin, Max on 2023/07/10.
 //
+
+var bankManager = BankManager(bank: Bank())
+
+bankManager.workBankManager()
```

Comment:
네이밍이 중복되어보여요.
개인적으로 코드는 영어 한 문장으로 자연스럽게 읽히는게 중요하다고 생각해요.
```suggestion
bankManager.start()
```
```suggestion
bankManager.work()
```
```suggestion
bankManager.open()
```

Reply:
`work()`로 변경했습니다🫡